src/core/include/openvino/op/is_inf.hpp

```diff
@@ -14,6 +14,7 @@ namespace v10 {
 class OPENVINO_API IsInf : public Op {
 public:
     OPENVINO_OP("IsInf", "opset10");
+    BWDCMP_RTTI_DECLARATION;
```

Comment:
Not sure that we need the `BWDCMP_RTTI_DECLARATION` in operators any more. This should be consulted with the broader team and possibly removed.

Reply:
As far I know, `BWDCMP_RTTI_DECLARATION` is not needed for new operations with Op as a base class (only in case of inheritance from older class with static type info member like UnaryElementwiseArithmetic).

Reply:
Thanks, `BWDCMP_RTTI_DECLARATION` and `BWDCMP_RTTI_DEFINITION` were removed.